packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1138,12 +1188,14 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     properties.add(new FlagProperty('isFocused', value: _hasFlag(SemanticsFlag.isFocused), ifTrue: 'focused'));
     properties.add(new FlagProperty('isButton', value: _hasFlag(SemanticsFlag.isButton), ifTrue: 'button'));
     properties.add(new FlagProperty('isTextField', value: _hasFlag(SemanticsFlag.isTextField), ifTrue: 'textField'));
+    properties.add(new FlagProperty('isInvisible', value: isInvisible, ifTrue: 'invisible'));
     properties.add(new StringProperty('label', _label, defaultValue: ''));
     properties.add(new StringProperty('value', _value, defaultValue: ''));
     properties.add(new StringProperty('increasedValue', _increasedValue, defaultValue: ''));
```

Comment:
Should this include the `nextNodeId` if set? I imagine it might be useful when you debug traversal order. And this is what gets printed out when you hit "S" during a hot run.

Reply:
Good point.  Added.